sdk/cognitivelanguage/Azure.AI.Language.Conversations/README.md

```diff
@@ -139,29 +113,6 @@ catch (RequestFailedException ex)
 
 You will notice that additional information is logged, like the client request ID of the operation.
 
-```text
```

Comment:
This should be kept. It's standard in all our SDK READMEs.

Reply:
The original message is about the "azure search service". I wanted to recreate the error for the conversations service using the `BadRequest` Test in Readme.cs. But I received a 401 error instead. I will bring back the original message.

Reply:
You could generate an error and copy that out.